verstack/stratified_continuous_split.py

```diff
@@ -121,41 +126,68 @@ def scsplit(X, y, test_size = 0.3, train_size = 0.7, continuous = True, random_s
     if random_state:
         np.random.seed(random_state)
 
+    if len(args) == 2:
+        X = args[0]
+        y = args[1]
+    else:
+        X = args[0].drop(stratify.name, axis = 1)
+        y = args[0][stratify.name]
```

Comment:
Are you sure this is going to work? 
If the default argument stratify = None, then if the user has not defined this argument - this first X & y definition is going to throw a TypeError.